packages/zwave-js/src/lib/telemetry/statistics.ts

```diff
@@ -55,6 +61,16 @@ export async function sendStatistics(
 		}
 		return false;
 	} catch (e) {
+		if (isObject(e.response) && e.response.status === 429) {
+			// We've hit the rate limiter. Figure out when we may try again.
+			if (
+				isObject(e.response.headers) &&
+				"retry-after" in e.response.headers
+			) {
+				const retryAfter = parseInt(e.response.headers["retry-after"]);
+				if (Number.isInteger(retryAfter)) return retryAfter;
+			}
+		}
 		// didn't work, try again later
 		return false;
```

Comment:
What about adding a log here in case of error? Maybe just ignore network errors

Reply:
That's logged at the call site anyways.